packages/metascraper-author/index.js

```diff
@@ -29,7 +30,7 @@ const wrap = rule => ({ htmlDom }) => {
 
 const strict = rule => $ => {
   const value = rule($)
-  return REGEX_STRICT.test(value) && value
+  return REGEX_STRICT.test(value) && !REGEX_DATE.test(value) &&value
```

Comment:
When tests run on travis, first it runs linter:
https://travis-ci.org/microlinkhq/metascraper/jobs/342081136#L3716

It fails beacuse a missing space after `&&` is necessary 🙂